.github/workflows/schedule_pg_almalinux8.yml

```diff
@@ -0,0 +1,33 @@
+---
+# yamllint disable rule:truthy
+
+name: scheduled PostgreSQL (AlmaLinux 8)
+
+on:
+  schedule:
+    - cron: '15 1 * * *'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: pip3 install molecule[docker] ansible
+
+      - name: Run Molecule tests
+        run: molecule test
+        env:
+          PY_COLORS: '1'
+          ANSIBLE_FORCE_COLOR: '1'
+          MOLECULE_DISTRO: almalinux8
```

Comment:
`geerlingguy/docker-almalinux8-ansible` image does not exist. 

For now, I can suggest adding tests for AlmaLinux to the TODO list. \
You can temporarily delete the schedule_pg_almalinux8.yml file.